lib/voxpupuli/test/rake/validate.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'metadata-json-lint/rake_task'
+require 'puppet-strings/tasks'
+
+desc 'Check syntax of Ruby files and call :syntax and :metadata_lint'
+task :validate do
```

Comment:
If we implement the checks below as tasks this can be
```suggestion
task validate: ['validate:ruby', 'syntax', 'metadata_lint', 'validate:strings']
```